workshop/content/20-typescript/70-advanced-topics/200-pipelines/5000-test-actions.md

```diff
@@ -87,32 +85,36 @@ export class WorkshopPipelineStack extends cdk.Stack {
         // PIPELINE CODE HERE...
 
         const deploy = new WorkshopPipelineStage(this, 'Deploy');
-        const deployStage = pipeline.addApplicationStage(deploy);
-        deployStage.addActions(new ShellScriptAction({
-            actionName: 'TestViewerEndpoint',
-            useOutputs: {
-                ENDPOINT_URL: //TBD
-            },
-            commands: [
-                'curl -Ssf $ENDPOINT_URL'
-            ]
-        }));
-        deployStage.addActions(new ShellScriptAction({
-            actionName: 'TestAPIGatewayEndpoint',
-            useOutputs: {
-                ENDPOINT_URL: //TBD
-            },
-            commands: [
-                'curl -Ssf $ENDPOINT_URL/',
-                'curl -Ssf $ENDPOINT_URL/hello',
-                'curl -Ssf $ENDPOINT_URL/test'
-            ]
-        }));
+        const deployStage = pipeline.addStage(deploy);
+
+        deployStage.addPost(
+            new CodeBuildStep('TestViewerEndpoint', {
+                projectName: 'TestViewerEndpoint',
+                envFromCfnOutputs: {
+                    ENDPOINT_URL: //TBD
+                },
+                commands: [
+                    'curl -Ssf $ENDPOINT_URL'
+                ]
+            }),
+
+            new CodeBuildStep('TestAPIGatewayEndpoint', {
+                projectName: 'TestAPIGatewayEndpoint',
+                envFromCfnOutputs: {
+                    ENDPOINT_URL: //TBD
```

Comment:
?

Reply:
It comes later in the steps. Look at the diff from before :)